includes/blocks/course-theme/class-course-navigation.php

```diff
@@ -55,10 +57,12 @@ public function __construct() {
 	 * Register course navigation block.
 	 */
 	private function register_block() {
-		Sensei_Blocks::register_sensei_block(
+		\Sensei_Blocks::register_sensei_block(
```

Comment:
Not sure about the good practices, but now that we added a namespace would make more sense to add these classes adding as `use` and use it without the `\`?

Reply:
I think generally it's more clarity to use full namespaces, and also makes the code easier to move. For these ones it's just top-level classes without a long namespace, each used 2-3 times, so it's actually more typing to add the `use` statements :)